setup.py

```diff
@@ -15,6 +15,40 @@
 version = metadata['version']
 
 
+class BumpCommand(Command):
+    """ Bump the __version__ number and commit all changes. """
+
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        version = metadata['version'].split('.')
+        version[-1] = str(int(version[-1]) + 1)  # Bump the final part
+
+        try:
+            print('old version: %s  new version: %s' %
+                  (metadata['version'], '.'.join(version)))
+            raw_input('Press enter to confirm, or ctrl-c to exit >')
+        except KeyboardInterrupt:
+            raise SystemExit("\nNot proceeding")
+
+        old = "__version__ = '%s'" % metadata['version']
+        new = "__version__ = '%s'" % '.'.join(version)
+
+        with open('ceph_installer/__init__.py', 'w') as fileh:
+            fileh.write(module_file.replace(old, new))
```

Comment:
Never a dumb question! :)

The problem with `module_file` is that it was meant to be used once. Without taking care of even properly closing the file after reading. So that variable (on line 12) could be a function/helper: 

    def read_module_contents():
        with open("ceph_installer/__init__.py") as installer_init:
            return installer_init.read()

    module_file = read_module_contents()

And then, here this method should ask for it *again*:

    module_file = read_module_contents()
    with open('ceph_installer/__init__.py', 'w') as fileh:
        fileh.write(module_file.replace(old, new))